nsxt/resource_nsxt_edge_transport_node.go

```diff
@@ -447,8 +447,20 @@ func getStandardHostSwitchSchema(nodeType string) *schema.Schema {
 					Computed:    true,
 				},
 				"host_switch_profile": getHostSwitchProfileIDsSchema(),
-				"ip_assignment":       getIPAssignmentSchema(false),
-				"ipv6_assignment":     getIPv6AssignmentSchema(),
+				"uplink_profile": {
```

Comment:
Should uplink_profile and vtep_ha_profile have a conflictsWith with hostSwitchProfile?

Reply:
Unfortunately `ConflictsWith` [does not work](https://github.com/hashicorp/terraform-plugin-sdk/blob/4d5e3b9a3211f139b41e78ea9ee880c8bf9bb87f/helper/schema/schema.go#L262) with nested schema.
This should be solved in plugin framework.